wordpress-mobile/WordPress-iOS
# Replace `MediaService.getMediaWithID` with the corresponding function in `MediaRepository`

What says in the title.

## Test Instructions

I tried to find the user flows that cover all code changes. But they are not easy to find. This user flow below verifies the changes in `MediaThumbnailCoordinator`. The rest of the changes are very similar, verifying this one flow probably is enough.

1. Log onto a site in WordPress.com and prepare a page that has a featured image.
1. Launch the app and sign out.
1. Sign in the site and go straight to My Sites -> Pages
2. Verify that the featured image can be displayed.

## Regression Notes
1. Potential unintended areas of impact
None.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
What's described in the Test Instructions section.

3. What automated tests I added (or what prevented me from doing so)
None.

PR submission checklist:


UI Changes testing checklist: N/A
